crates/jstz_proto/src/executor/fa_deposit.rs

```diff
@@ -21,11 +22,11 @@ const FA_DEPOSIT_GAS_LIMIT: usize = usize::MAX;
 const NULL_ADDRESS: &str = "tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx";
 const DEPOSIT_URI: &str = "/-/deposit";
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct FaDepositReceiptContent {
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct FaDepositReceipt {
     pub receiver: PublicKeyHash,
```

Comment:
Just a comment: no need to get everything this in this PR, but we might want to write descriptions/docstrings for (most) fields in structs with `ToSchema` derived.

Reply:
Agreed that it would be useful since the descriptions would be pushed down into the client libs too. Tracked the work in: https://linear.app/tezos/issue/JSTZ-188/add-description-for-schema-object-fields